W-12460726-add-limits (#98)
mulesoft/docs-hosting: cloudhub-2/modules/ROOT/pages/ch2-limits.adoc

Application size:: Up to 200 MB

Custom TLS contexts per private space:: 10
+
*Maximum number of Client Certificates in a TLS Context:*
+

> There is no limit on the number of client certificates, but this is limited by the store file size.

Reviewer: ```suggestion
There is no limit on the number of client certificates, but there is a limit on the store file size.
```

+

> A TLS Context can reference 1 keystore and 1 truststore, which have their respective file size limits:

Reviewer: ```suggestion
A TLS Context can reference one keystore and one truststore, which have their respective file size limits:
```
